app/views/spree/admin/orders/invoice2.html.haml

```diff
@@ -8,7 +8,7 @@
           = t :tax_invoice
       - if @order.distributor.display_invoice_logo? && @order.distributor.logo.variable?
         %td{ :align => "right", rowspan: 2 }
-          = wicked_pdf_image_tag @order.distributor.logo.variant(resize_to_limit: [150, 150])
+          = wicked_pdf_image_tag @order.distributor.logo_url(:small)
```

Comment:
This is just a quick fix because the original image was 150 pixels while the new one is now 180 pixels. But maybe that's okay anyway? 150 pixels is very small. Let's keep it this way and save on image variants.

Reply:
Yep, I thought this quick fix was enough.